### DISCUSSION TITLE
Charger schematics in PineWiki outdated

### DISCUSSION THREAD

**Discussion by sixtyfive** [pine64/OpenPineBuds#63] [Q&A]
Hi, I know you're not Pine64, but do you happen to have access to the hardware schematics for the charger as it is being shipped? That is, without wireless charging capability and hence without CP2021 IC? The wiki only has the _old_ schematics _with_ the CP2021...

**Comment by gamelaster** [comment]
I am not fully sure, but isn't it just about the fact, that CP2021 and other parts are just not mounted, else it is same schematics? If this is not the case, I can ask Pine Store for giving us latest schematics.

**Reply by sixtyfive** [reply]
Alright, so I take the CP2021 thing back; you're right, the footprints are there for it and its chicken food. Other assorted items thus far: CH347T instead of CH342K. Only TP5 exists. Possibly LP3992 swapped for a similar one, but difficult to read. The differences are unfortunately not confidence-inspiring for a repair attempt...

**Reply by gamelaster** [reply]
Hi @sixtyfive , thanks. I asked Pine Store for updated schematics. I will let you know when it will be available.
--- gamiee
